.gitbook/3-protocol-devs/1-dep-specs/5-confidential-oracle.md

```diff
@@ -0,0 +1,57 @@
+# Confidential Oracle
+
+- Status: Draft
+- Created: 2023-01-06
+- Modified: 2023-01-06
+- Authors
+    - Inchul Song <icsong@medibloc.org>
+    - Youngjoon Lee <yjlee@medibloc.org>
+    - Gyuguen Jang <gyuguen.jang@medibloc.org>
+    - Hansol Lee <hansol@medibloc.org>
+    - Myongsik Gong <myongsik_gong@medibloc.org>
+
+## Synopsis
+
+This document explains that why the oracles must use confidential computing.
+
+## Motivation
+
+In DEP, the data must be verified as explained in the [data validation documentation](./4-data-validation.md).
+Since the data is sensitive, we designed the off-chain oracle. Verifying data means that the content of data could be
+shown to the oracle.
+It also means that the oracle operators(human) can see the content of sensitive data. For example, if the oracle
+software stores the important sensitive data to the disk or storage without sealing it,
+it can be read by anyone who can access the disk or storage. Plus, if the malicious oracle operator modified the source
+code of the oracle to print logs of sensitive data or to leak a contents somewhere else,
+it would cause the serious privacy problems.
```

Comment:
IMO, it seems too detailed in motivation.
I think just need to write why confidential computing is used here.
We can move this details into the main contents.
```suggestion
```